src/api/utils.ts

```diff
@@ -0,0 +1,17 @@
+import { API_URL } from './constants';
+
+export const getData = <T>(url: string): Promise<T> => fetch(url).then(response => response.json());
+export const getPosts = (): Promise<PostInterface[]> => getData<PostInterface[]>(`${API_URL}posts`);
+export const getUsers = (): Promise<UserInterface[]> => getData<UserInterface[]>(`${API_URL}users`);
+export const getComments = (): Promise<CommentInterface[]> => getData<CommentInterface[]>(`${API_URL}comments`);
+export const getPreparedPosts = async () => {
+  const posts: PostInterface[] = await getPosts();
+  const users: UserInterface[] = await getUsers();
+  const comments: CommentInterface[] = await getComments();
+
+  return (posts.map(post => ({
```

Comment:
do you really need those braces after return?

Reply:
yes, without the obvious return i need both braces